src/controllers/hello-world.api.ts

```diff
@@ -1,11 +1,17 @@
 // Copyright IBM Corp. 2017. All Rights Reserved.
-// Node module: @loopback/loopback-next-hello-world
+// Node module: loopback-next-hello-world
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 'use strict';
 
 export const controllerSpec =
 {
+  "swagger": "2.0",
+  "basePath": "/",
+  "info": {
+    "title": "LoopBack Application",
+    "version": "1.0.0",
+  },
```

Comment:
Controller specifications don't need swagger preamble, they have only optional `basePath` and required `paths` properties.